about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>about</title>
+    <script src="https://use.fontawesome.com/54aef7fad9.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <div class="top">
+            <ul class="top-header">
+                <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+                <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <nav class="navbar">
+            <i class="fa fa-bars" aria-hidden="true"></i>
+            <a href="index.html"><img src="images/logo.png" alt="Logo" class="logo"></a>
+            <ul class="nav-links">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">JSI Conference</button></li>
+            </ul>
```

Comment:
- [ ] In desktop navbar let's compare and try to make its `element positions` the same as the original 👇 
-Yours 👇 
![navbar-st](https://user-images.githubusercontent.com/67087664/147964206-9f80a582-87a2-493c-8294-09e274068b98.png)
- Original 👇 
![navbar](https://user-images.githubusercontent.com/67087664/147964269-4ef1a05b-108d-4df5-a7c4-68ac35a3a9f8.png)